x-pack/docs/en/monitoring/indices.asciidoc

```diff
@@ -11,6 +11,8 @@ You can retrieve the templates through the `_template` API:
 ----------------------------------
 GET /_template/.monitoring-*
 ----------------------------------
+// CONSOLE
+// TEST[skip:404 error]
```

Comment:
Will `// TEST[catch:missing]` do it?

Reply:
Or we could make sure that the templates are created first.

Reply:
Thanks @nik9000, catch:missing worked.  Can we document or link to that list of keywords somewhere?

Reply:
https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/test/README.asciidoc#required-operators has a list that should be up to date. I tend to look [here](https://github.com/elastic/elasticsearch/blob/master/test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/DoSection.java#L336-L351) though.

Reply:
We could probably do with a link from https://github.com/elastic/elasticsearch/blob/master/docs/README.asciidoc though.